base/strings/util.jl

```diff
@@ -400,6 +400,7 @@ single character, a vector or a set of characters, a string, or a regular expres
 is a function, each occurrence is replaced with `r(s)` where `s` is the matched substring.
 If `pat` is a regular expression and `r` is a `SubstitutionString`, then capture group
 references in `r` are replaced with the corresponding matched text.
+To delete patterns in `pat` set `r` to an empty String (`""`).
```

Comment:
Thanks @shorty66! :) Perhaps "To [remove|delete] instances of `pat` from `string`, set `r` to the empty `String` (`""`)." would be slightly clearer?